src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    background-image: url("../images/star.svg");
+    min-width: 16px;
+    min-height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+}
+
+.stars .stars__star:nth-child(-n + 4) {
+  background-image: url("../images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
please check the checklist
![image](https://user-images.githubusercontent.com/22824431/222739602-43050e88-451e-4720-9776-64f3c7f71f03.png)